gui/CameraView.cxx

```diff
@@ -580,11 +580,13 @@ void CameraView::setLandmarksData(kwiver::vital::landmark_map const& lm)
   }
 
   auto fields = QHash<QString, FieldInformation>{};
-  fields.insert("Elevation", {Elevation, {minZ, maxZ}});
```

Comment:
not as written, but `insert("Elevation", FieldInformation{Elevation, {minZ, maxZ}});` seems to work.  I'll revise tomorrow.